### ISSUE TITLE
ElasticSearch Watcher Score Fix

### ISSUE THREAD

**Issue by mikegrima** [Netflix/security_monkey#283]
Need to make a quick fix to the ElasticSearch Watcher so that if a domain permits the `root` user of an account, it only raises an issue if it is granting `root` from a _different_ account.

The logic fix here can be adapted to S3 (will submit another issue for that feature once this one is complete).
